module/VuFind/src/VuFind/Service/MarkdownFactory.php

```diff
@@ -63,10 +63,27 @@ class MarkdownFactory implements FactoryInterface
     public function __invoke(
         ContainerInterface $container, $requestedName, array $options = null
     ) {
+        $config = $container->get(\VuFind\Config\PluginManager::class)
+            ->get('markdown')->Markdown;
         return new GithubFlavoredMarkdownConverter(
             [
-            'html_input' => 'strip',
-            'allow_unsafe_links' => false,
+                'html_input' => $config->html_input ?? 'strip',
+                'allow_unsafe_links' => $config->allow_unsafe_links ?? false,
+                'enable_em' => $config->enable_em ?? true,
+                'enable_strong' => $config->enable_strong ?? true,
+                'use_asterisk' => $config->use_asterisk ?? true,
+                'use_underscore' => $config->use_underscore ?? true,
+                'unordered_list_markers' => isset($config->unordered_list_markers)
+                    && is_array($config->unordered_list_markers)
```

Comment:
Did you test that this setting works as expected? I would expect `$config->unordered_list_markers` to be a `Zend\Config\Config` object in this context, whether or not it holds multiple values, so I'm concerned this check may never evaluate correctly. I haven't tried it, though, so perhaps the `Config` object is smarter than I think!

Reply:
Thanks for catching this. Config object can't be as smart due to PHP limitations (is_array always return false on ArrayAccess). I fixed that in folowup.

Reply:
Glad to help!